### ISSUE TITLE
Summary

### ISSUE THREAD

**Issue by jmcrey** [frankja13/dvp-u6-netflix-data#1]
## Rubric Score
 
### Criteria 1: Pandas and CSV
* _Score Level:_ 4 (Exceeds Expectations)
* _Comment(s):_ The notebook imports the necessary libraries and converts the CSV to a Pandas DataFrame.
 
### Criteria 2: Implementation of visualizations in spec
* _Score Level:_ 4 (Exceeds Expectations)
* _Comment(s):_ The notebook renders all Seaborn and Matplotlib visualizations and content provided in the design specification.
 
### Criteria 3: Color
* _Score Level:_ 4 (Exceeds Expectations)
* _Comment(s):_ Color carries meaning and visualizations use different palettes with purpose. A highlight is that each of the graphs use various colors to display their purpose; some even use different line styles and pallets entirely. Great use of color!
 
### Criteria 4: Style
* _Score Level:_ 2 (Approaches Expectations)
* _Comment(s):_ Some chart labels and titles are custom, others are unchanged. Of the ones that are custom, they are easy to read and succinct. They also accurately describe the graph. However, some (e.g. the first graph) do not contain a title, which makes the data hard to understand. 

### Criteria 4: Presentation and framing of visualization
* _Score Level:_ 3 (Meets Expectations)
* _Comment(s):_ Presentation cohesively presents every chart and has an appropriate description for each. Also, this presentation was beautifully formatted and gave the charts a lovely display, which made them just a pleasure to look at. Furthermore, the description alongside the chart accurately describe the intent of the data in the chart. So, overall, this was a lovely presentation. 
 
### Overall Score: 17/20

Overall, this was a solid project that correctly displayed and describe each of the required charts. The biggest note of improvement is to ensure to include a title for every chart, and to include some analysis on the slides of the presentation. While the description is useful, it does not give the reader everything they would want to know about the chart. So, instead of leaving the insights in the comments of the presentation, it might be useful to include it inside the actual slides of the presentation. Furthermore, a title can usually suffice for the description of the graph; so, it would be better to replace the description with the useful insights gleamed from the displayed chart. 

That being said, this project accurately implemented and display every chart, made great use of color, correctly imported each package, and presented a beautiful presentation. Great job!

**Comment by frankja13** [comment]
Yes! I am very happy and I will use these comment to improve my work. Thank you very much!
